packages/@orchidui-vue/src/DataDisplay/ListItem/components/OcTerminal.vue

```diff
@@ -15,15 +19,24 @@ defineProps({
   },
   isDisabled: Boolean,
 });
+
+const imageSizes = {
+  small: "w-10",
+  default: "w-[56px]",
+  big: "w-16",
```

Comment:
size should be `small medium large` 
does it work in class binding with tailwind?
```suggestion
  small: "w-10",
  medium: "w-[56px]",
  large: "w-16",
```

Reply:
@HoaiLee yes it's working